package.json

```diff
@@ -13,6 +13,7 @@
     "reshape": "^0.4.0",
     "reshape-loader": "^0.4.0",
     "rooftop-client": "^0.2.1",
+    "spike-util": "^0.4.0",
```

Comment:
Yeah, so spike is on v1 now. You can run `yarn upgrade-interactive` to get all your dependencies up to date!